src/Locale/it_IT/default.po

```diff
@@ -33,10 +33,10 @@ msgid "All Types"
 msgstr "Tutti i tipi"
```

Comment:
Original labels should be in english... why do we have **"Tutti i tipi"** in this .po ?
There are others like **"Tutti gli oggetti"** below or **"Azioni sugli oggetti selezionati"** above

Reply:
wrong strings in templates... I'll give it a look

Reply:
```
msgid "Actions on selected items"
msgstr "Azioni sugli oggetti selezionati"
[...]
msgid "All Types"
msgstr "Tutti i tipi"
[...]
msgid "All objects"
msgstr "Tutti gli oggetti"
```
they seem fine to me. @stefanorosanelli